pom.xml

```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
```

Comment:
We have changed the company name to WSO2 LLC.
Please refer this email - "License Header for WSO2 code"

Reply:
Updated